file-formats/aifi/type/zif_aff_aifi_v1.intf.abap

```diff
@@ -0,0 +1,496 @@
+      "! Interface display name
+      display_name                  TYPE c LENGTH 120,
+      " (type /aif/msg_lifetime)
+      "! <p class="shorttext">Application Log Lifetime</p>
+      "! Application log lifetime
+      "! $default '90'
+      app_log_lifetime              TYPE n LENGTH 10,
+      " (type /aif/ifdirection)
+      "! <p class="shorttext">Direction</p>
+      "! Direction
+      direction                     TYPE ty_direction_type,
+      " /AIF/PRE_PROCESSING
+      "! <p class="shorttext">Preprocessing</p>
+      "! Preprocessing
+      uses_preprocessing            TYPE abap_bool,
+      " /AIF/PROXY_XML_TRANSFORMATION
+      "! <p class="shorttext">Proxy XML Transformation</p>
+      "! Use proxy xml transformation
+      uses_proxy_xml_transformation TYPE abap_bool,
+      " /AIF/PRX_XXML
+      "! <p class="shorttext">Extend XML Handling</p>
+      "! Extend xml handling
+      uses_extended_xml_handling    TYPE abap_bool,
+      "! <p class="shorttext">Postprocessing</p>
+      "! Postprocessing
+      uses_postprocesssing          TYPE abap_bool,
+    END OF ty_general_information.
+
+  TYPES:
+    "! <p class="shorttext">Recipient</p>
+    "! Recipient
+    BEGIN OF ty_recipient,
+      " (type /aif/recipient_object_name)
+      "! <p class="shorttext">Alert Recipient</p>
+      "! Alert recipient
+      "! $required
+      recipient TYPE c LENGTH 40,
+    END OF ty_recipient.
+
+  "! <p class="shorttext">Recipients</p>
+  "! Recipients
+  TYPES ty_recipients TYPE STANDARD TABLE OF ty_recipient WITH DEFAULT KEY.
+
+  TYPES:
+    "! <p class="shorttext">General</p>
+    "! General key field settings
+    BEGIN OF ty_key_field_settings,
+      " (type /aif/key_fieldname, field of index table)
+      "! <p class="shorttext">Key Field Name</p>
+      "! Key field name
+      "! $required
+      key_field_name     TYPE zif_aff_types_v1=>ty_object_name_30,
+      " (type rollname)
+      "! <p class="shorttext">Data Element</p>
+      "! Data element
+      "! $required
+      data_element       TYPE zif_aff_types_v1=>ty_object_name_30,
+      " (type /aif/stexti)
+      "! <p class="shorttext">Name of Select-Option/Parameter</p>
+      "! Name of select-option/parameter
+      select_option_name TYPE c LENGTH 8,
+      " (type /aif/lfieldname_kflds, field of sap/raw structure)
+      "! <p class="shorttext">Field Name</p>
+      "! Field name in define key fields
+      field_name         TYPE c LENGTH 120,
+      " (type /aif/raw_or_sap_kflds)
+      "! <p class="shorttext">SAP or RAW Structure</p>
+      "! Is sap or raw structure?
+      sap_raw_structure  TYPE ty_sap_raw_structure_type,
+      " /AIF/FLD_IS_SELECT_OPT
+      "! <p class="shorttext">Select-Option</p>
+      "! Is select-option?
+      is_select_option   TYPE abap_bool,
+      " /AIF/IS_COL
+      "! <p class="shorttext">Display as Column</p>
+      "! Display as column
+      "! $default 'X'
+      is_column_display  TYPE abap_bool,
```

Comment:
We are a bit unsure again about `is_column_display`. Does it have the same meaning as `Display as Column`? What do you think? Maybe `display_as_column`?

Reply:
This indicates whether the field is displayed as a column or not displayed at all. You cannot display it in an alternative fashion. I changed the description a little bit.